JASP-Engine/JASP/R/common.R

```diff
@@ -2955,3 +2955,7 @@ postProcessModuleInstall <- function(moduleLibraryPath)
     .postProcessLibraryModule(allFiles)
   }
 }
+
+.setSeedJASP <- function(options) {
```

Comment:
Okay, since we're gonna use this in many places, let's make this a bit more developer-friendly by adding some error messages:
```r
.setSeedJASP <- function(options) {
  
  if (is.list(options) && c("setSeed", "seed") %in% names(options)) {
    if (isTRUE(options[["setSeed"]]))
      set.seed(options[["seed"]])
  } else {
    stop(paste(".setSeedJASP was called with an incorrect argument.",
               "The argument options should be the options list from QML.",
               "Ensure that the SetSeed{} QML component is present in the QML file for this analysis."))
  }
}
```

See the other comment for an explanation of the "SetSeed{} QML component"

Reply:
Yes, changed, thank you.